public/js/components/modal.jsx

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var React = require('react');
+var gettext = require('utils').gettext;
+var cx = require('classnames');
+
+
+module.exports = React.createClass({
+
+  displayName: 'Modal',
+
+  propTypes: {
+    children: React.PropTypes.array.required,
```

Comment:
Yeah, this is fixed in my new branch. It should also be node not array or object.